node_modules/oae-ui/lib/api.js

```diff
@@ -1125,12 +1125,71 @@ var renderTemplate = module.exports.renderTemplate = function(template, data, lo
              */
-            'parse': url.parse
+            'parse': url.parse,
+
+            /**
+             * Escape all characters except the following: alphabetic, decimal digits, - _ . ! ~ * ' ( )
+             *
+             * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
+             */
+            'encode': encodeURIComponent,
+
+            /**
+             * Ensure that a link is an absolute URL. If a relative link is
+             * passed in, it will be prefixed with the base url.
+             *
+             * @param  {String}     link        The link to check
+             * @param  {String}     baseUrl     The base url that can be used to prefix relative urls
+             * @return {String}                 The absolute link prefixed with the base url
+             */
+            'ensureAbsoluteLink': function(link, baseUrl) {
+                // If the link is empty or null, we return the empty string. This can happen when
+                // we try to link a private user (private users are scrubbed and have no profile path)
+                if (!link) {
+                    return '';
+
+                // If the link already has `http` in it (e.g., twitter profile pics) we return as-is
+                } else if (link.indexOf('http') === 0) {
+                    return link;
+
+                // Otherwise we prefix it with the base url
+                } else {
+                    return baseUrl + link;
+                }
+            },
+
+            /**
+             * Ensure that each link in an HTML fragment is an abolute url, If a relative link is
+             * found, it will be prefixed with the base url.
+             *
+             * @param  {String}     str         The html string in which to check for absolute links
+             * @param  {String}     baseUrl     The base url that can be used to prefix relative urls
+             * @return {String}                 The html in which each link is absolute
+             */
+            'ensureAbsoluteLinks': function(str, baseUrl) {
+                var html = $('<div>' + str + '</div>');
+                html.find('a').each(function(i, elem) {
+                    var link = $(this).attr('href');
+                    link = data.util.url.ensureAbsoluteLink(link, baseUrl);
+                    $(this).attr('href', link);
+                });
+                return html.html();
+            }
         }
     };
 
     // Parse and process it with Underscore
     try {
+        // Support <% include path/to/other/template.jst %>
+        template = template.replace(/<%\s*include\s*(.*?)\s*%>/g, function(match, path) {
+            if (fs.existsSync(path)) {
+                return fs.readFileSync(path, 'utf8');
+            } else {
+                log().warn({'path': path}, 'Could not found an underscore template');
+                return '';
+            }
+        });
```

Comment:
Wondering if this is something we should ignore, versus letting the server start up with knowingly bogus email templates?